doc/connectivity/resting_state_networks.rst

```diff
@@ -91,3 +95,74 @@ Finally, we can plot the map for different ICA components separately:
    displayed on your computer might not match those of the documentation. For
    a fair representation, you should display all components and
    investigate which one resemble those displayed above.
+
+Beyond ICA : Dictionary learning
+================================
+
+Recent work has shown that dictionary learning based techniques outperform
+ICA in term of stability and constitutes a better first step in a statistical
+analysis pipeline.
+Dictionary learning in neuro-imaging seeks to extract a few representative
+temporal elements along with their sparse spatial loadings, which constitute
+good extracted maps.
+
+.. topic:: **References**
+
+   * Gael Varoquaux et al. `Multi-subject dictionary learning to segment an atlas of brain spontaneous activity <http://hal.inria.fr/inria-00588898/en/>`_,
+     IPMI 2011, pp. 562-573, Lecture
+     Notes in Computer Science
+
+Applying DictLearning
+---------------------
+
+:class:'DictLearning' is a ready-to-use class with the same interface as CanICA.
+Sparsity of output map is controlled by a parameter alpha: using a
+larger alpha yields sparser maps.
+
+.. literalinclude:: ../../examples/connectivity/plot_compare_resting_state_decomposition.py
+    :start-after: # Dictionary learning
+    :end-before:# CanICA
```

Comment:
Genuine error in CircleCI, it looks like you need a space after the :

Try to run the doc generation with make html-strict and you'll be able to reproduce the error and possibly find other problems.